### PR TITLE
Changed navigation link of cloud icon to collate website

### PR DESCRIPTION
**Pull request by aniketkatkar97**
Changed redirection link for cloud icon in nav-bar to collate website

https://user-images.githubusercontent.com/51777795/205676204-7ab14002-ccab-4e78-bce6-7038c2bce320.mov

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@aniketkatkar97 , we have to change the link in both places.

image

**Comment by aniketkatkar97** [comment]
@Sachin-chaurasiya I added the link for the tile as well. Can you please review?

File: components/navigation/header.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const Header = () => {
             </a>
             <a
               className="btn fw-500 btn-primary rounded-pill"
-              href="https://share.hsforms.com/1fstvMCeZRZKTYA4nG1VTPgcq0j9"
+              href="https://www.getcollate.io/"
```
Comment (Sachin-chaurasiya) [inline components/navigation/header.js:89]: Can we take this link from constant, so in future we don't have to make changes in multiple files?

Comment (aniketkatkar97) [reply components/navigation/header.js:89]: Sure

Comment (aniketkatkar97) [reply components/navigation/header.js]: Done

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**